src/components/SweetsList/SweetsListStyles.ts

```diff
@@ -18,6 +18,7 @@ export const StyledMain = styled.main`
   & p {
     padding: 2rem;
     font-size: 4rem;
```

Comment:
You can also consider adding a standard set of spacing (padding/margin) values and / or font-sizes
just as a (rough) example something like: 
spacings: {
  small: 0.5rem,
  medium: 1rem,
  big: 2rem
 ......
}